Type error when using `"moduleResolution": "Node16"` and ESM

**Describe the bug**

```bash
 error TS2351: This expression is not constructable.
  Type 'typeof import("/Users/***/node_modules/@classmethod/athena-query/dist/index")' has no construct signatures.

5 const athenaQuery = new AthenaQuery(athena);
```

**To Reproduce**

1. Open `tsconfig.json`
2. Settings some value
    ```
    "compilerOptions": {
      "moduleResolutions": "node16",
      "module": "node16"
    }
    ```
4. Change file extension to `.mts`
5. See error

Package versions
- `@classmethod/athena-query`: 1.2.0
- `@aws-sdk/client-athena`: 3.540.0
- `typescript`: 5.4.3

**Expected behavior**
No error

**Screenshots**
None

**Desktop (please complete the following information):**

- OS: [e.g. iOS]
- Browser [e.g. chrome, safari]
- Version [e.g. 22]

**Smartphone (please complete the following information):**

- Device: [e.g. iPhone6]
- OS: [e.g. iOS8.1]
- Browser [e.g. stock browser, safari]
- Version [e.g. 22]

**Additional context**
None

--- a/package.json
+++ b/package.json
@@ -30,9 +30,14 @@
   ],
   "exports": {
     ".": {
-      "types": "./dist/index.d.ts",
-      "require": "./dist/index.cjs",
-      "import": "./dist/index.mjs"
+      "require": {
+        "types": "./dist/index.d.cts",
+        "default": "./dist/index.cjs"
+      },
+      "import": {
+        "types": "./dist/index.d.mts",
+        "default": "./dist/index.mjs"
+      }
     }
   },
   "scripts": {

